documentation/src/main/asciidoc/topics/con_eviction_custom_types.adoc

```diff
@@ -0,0 +1,19 @@
+[id='eviction_custom_types-{context}']
+= Custom Types with Memory-Based Eviction
+Using custom types with memory-based eviction requires specific configuration
```

Comment:
Do we normally call this custom types? Maybe user types or user class instances or something like that?